steering_controllers_library/src/steering_controllers_library.yaml

```
@@ -5,16 +5,9 @@ steering_controllers_library:
     description: "Timeout for controller references after which they will be reset. This is especially useful for controllers that can cause unwanted and dangerous behaviour if reference is not reset, e.g., velocity controllers. If value is 0 the reference is reset after each run.",
   }
 
-  front_steering: {
-    type: bool,
-    default_value: true,
-    description: "Is the steering on the front of the robot?",
-    read_only: true,
-  }
-
-  rear_wheels_names: {
+  wheels_names: {
```

```suggestion
  traction_wheels_names: {
```
or `traction_joints_names`